src/utils/hooks/useKubevirtUserSettings/useKubevirtUserSettings.ts

```diff
@@ -44,12 +52,26 @@ const useKubevirtUserSettings: UseKubevirtUserSettings = (key) => {
     fetchUserName();
   }, []);
 
+  useEffect(() => {
+    if (userName && !isAdmin) {
```

Comment:
u always try to create to a non admin user before knowing is the other user has default access. i think it will be better to first check if the user has default access , if not trying to create on is own space

Reply:
The solution to creating a project for each user is not relevant anymore, can you please review it again? :)